### PR TITLE
New evm chains

### PR DESCRIPTION
**Pull request by defisaur**
https://github.com/trustwallet/atlas/issues/1404

### CODE REVIEW

File: coin/coins.yml [hunk 1]
```diff
@@ -92,7 +92,7 @@
   name: IoTeX Network Mainnet
   decimals: 18
   blockTime: 10000
-  blockchain: IoTeX Network Mainnet
+  blockchain: Ethereum
```
Comment (iCell) [inline coin/coins.yml:95]: I'm not sure,  shouldn't add a new Iotex chain rather than changing the current chain?

Comment (covain) [reply coin/coins.yml:95]: Should be a new chain

Comment (covain) [reply coin/coins.yml:95]: Should be iotexevm with a coin id 10004689
 https://github.com/trustwallet/wallet-core/blob/master/registry.json#L1735

Comment (covain) [reply coin/coins.yml:95]: @iCell this change is correct. There is another iotex coin already exists in this file.

**Review by covain — APPROVED** [review]

File: coin/coins.yml (hunk 1 above)
Comment (defisaur) [reply coin/coins.yml:95]: @iCell yes we have iotex and iotex evm, previously I added iotex evm incorrectly, now fixed

**Pull request merged**